cmd/utils/utils.go

```diff
@@ -263,46 +280,61 @@ func register(logger *logrus.Entry, eth interfaces.Ethereum, cmd *cobra.Command,
 
 	// Contract orchestration
 	// Approve tokens for staking
-	txn, err := c.StakingToken().Approve(txnOpts, c.ValidatorsAddress(), big.NewInt(1_000_000))
-	if err != nil {
-		logger.Errorf("StakingToken.Approve() failed: %v", err)
-		return 1
-	}
-	rcpt, err := eth.Queue().QueueAndWait(ctx, txn)
-	if err != nil {
-		logger.Errorf("StakingToken.Approve() failed: %v", err)
-		return 1
-	}
-	if rcpt != nil && rcpt.Status != 1 {
-		logger.Errorf("StakingToken.Approve() failed")
+	var retry bool = true
+	for retry {
+		txn, err := c.StakingToken().Approve(txnOpts, c.ValidatorsAddress(), big.NewInt(1_000_000))
```

Comment:
no magic values, resolve from contracts

Reply:
this is going to be deleted soon because we'll be using the StakeNFT contract for this logic